doc-generator/doc_formatter/doc_formatter.py

```diff
@@ -1662,8 +1662,6 @@ def _parse_single_property_info(self, schema_ref, prop_name, prop_info, prop_pat
                         descr = descr + ' ' + combined_prop_item['longDescription']
```

Comment:
You probably want to remove this line as well -- same change, but for long descriptions.